Packs/RemoteAccess/Integrations/RemoteAccessv2/README.md

```diff
@@ -28,6 +29,12 @@ For login using root:
 - set `PermitRootLogin` to `yes`
 - set `PasswordAuthentication` to `yes`
 2. Restart the sshd server: `service sshd restart`
+
+## Configure the instance with SSH certificate
+Currently, the only type of certificate which is supported is the RSA private keys (.PEM) files.
+In case access is required to an instance in the cloud, make sure to add to the credentials object the content
```

Comment:
your sentence sounds a little backwards to me here. Maybe something like:
`In case access is required to an instance in the cloud, make sure to add the content of the PEM file provided by the cloud provider to the credentials object provided to the instance.`

Is this only true when its in the cloud though? Can we get rid of the first part? Or are you trying to highlight that in this case the cloud provider should be providing the pem?

Reply:
I am trying to highlight here the fact that the in case where the user is trying to access to an instance in the cloud, that he should get it from the cloud provider. So yes, that is only true in case the user tries to access into a cloud instance. In cases where the instance is on its remote network, then i expect him to be configure a PEM file of his own (probably using ssh-keygen)

Reply:
So how about this?
`In case access is required to an instance in the cloud, use the PEM file provided by the cloud provider`
We already said what to do with a key once you have it

Reply:
sounds good, added the following: 
```
In case access is required to an instance in the cloud, use the PEM file provided by the cloud provider to access the instance.
```